src/text/dialog_2.asm

```diff
@@ -848,4 +848,4 @@ Dialog106::
 
 Dialog107::
     db "You've got the  "
-    db "Thunder Drum!@"
\ No newline at end of file
+    db "Thunder Drum!@"
```

Comment:
PLease add new line at the end back in again. In gitlab there is then a text: "no newline" that is not part in the real document. I do not know if this is the case for github as well, but it looks better in the editor as well. But this is just my opinion.

Reply:
Yup ! The new line was actually missing before, and is now correctly present :)